### ISSUE TITLE
Contact search button doesn't have an accessible label

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1983]
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/2501adbc-acfd-48f7-aedd-cbb251730034)

**Comment by cthomasONRR** [comment]
@Maroyafaied @R-mohansai Since this issue was created, we changed the contacts search:
![image](https://user-images.githubusercontent.com/72526208/204352582-c01dec06-9012-42b2-bd21-93a5a7bbf8c1.png)
I ran the Lighthouse check on this, and it is NOT giving an accessibility error.

I propose we close this issue and work on #1981 instead.
